boards/aitewinrobot-esp32c3-supermini/board.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Aitewin Robot ESP32-C3 SuperMini",
+  "version": 1.8,
+  "description": "A common ESP32 dev board by AITEWIN ROBOT (Derived from Adafruit QT Py ESP32-C3)",
+  "author": "GeekGarage",
+  "mcu": "esp32-c3",
+  "fqbn": "esp32:esp32:esp32c3",
+  "width": 18.00,
+  "height": 24.576,
+  "pins": {
+    "5V": { "x": 16.702, "y": 3.238, "target": "power(5)" },
+    "GND": { "x": 16.702, "y": 5.778, "target": "GND" },
+    "3V3": { "x": 16.525199, "y": 8.318, "target": "power(3.3)" },
+    "4/GPIO4/A4/SCK": { "x": 16.702, "y": 10.858, "target": "GPIO4" },
+    "3/GPIO3/A3": { "x": 16.702, "y": 13.398, "target": "GPIO3" },
+    "2/GPIO2/A2": { "x": 16.702, "y": 15.938, "target": "GPIO2" },
+    "1/GPIO1/A1": { "x": 16.702, "y": 18.478, "target": "GPIO1" },
+    "0/GPIO0/A0": { "x": 16.702, "y": 21.018, "target": "GPIO0" },
+    "5/GPIO5/A5/MISO": { "x": 1.462, "y": 3.238, "target": "GPIO5" },
+    "6/GPIO6/MOSI": { "x": 1.462, "y": 5.778, "target": "GPIO6" },
+    "7/GPIO7/SS": { "x": 1.462, "y": 8.318, "target": "GPIO7" },
+    "8/GPIO8/SDA": { "x": 1.462, "y": 10.858, "target": "GPIO8" },
+    "9/GPIO9/SCL": { "x": 1.462, "y": 13.398, "target": "GPIO9" },
+    "10/GPIO10": { "x": 1.462, "y": 15.938, "target": "GPIO10" },
+    "20/GPIO20/RX": { "x": 1.462, "y": 18.478, "target": "GPIO20" },
+    "21/GPIO21/TX": { "x": 1.462, "y": 21.018, "target": "GPIO21" }
```

Comment:
Let's keep the pin names simple. I'm thinking:

1. For "RX/TX" pins, call them as "RX" / "TX"
2. For all the other pins, just the GPIO index (so "4", "3", etc)

This will keep us consistent with the other ESP32 boards (such as the esp32-devkit-c-v4)

Reply:
While I see your point in relation to other boards, and i will change it, I don't agree at all with simplifying the names.
My reasoning being when you run wires between devices, it can be hard to remember what pin is what and have what functionality, and then you would constantly have to refer back to the diagram. So while i see the aesthetically pleasing by simplifying pin names, you also make it a much worse user experience, by prioritizing form over function.

Let me know if you still think this should be changed and I'll fix it

Reply:
oh and as a tablet and PC user, i can say that prioritizing the form over function, is just compounded when working on a tablet with a single screen and you have to flip flop between tabs to see the functions of the pins. The user experince just gets even worse, then on top of that, add in if you are a new user to all of this, both arduino and Wokwi, you have enough information to keep track on and start learning.

Reply:
I've simplyfied the pin name, but i would like you to consider my points about user experience issues